app/build.gradle

```diff
@@ -836,3 +836,6 @@ android.applicationVariants.all { variant ->
         dependsOn "package${variant.name.capitalize()}"
     }
 }
+
+// Enable expiration by major version.
+ext.gleanExpireByVersion = 1
```

Comment:
This will expire all your metrics: you need to provide the major version of Fenix here. See the [Glean docs](https://mozilla.github.io/glean/book/language-bindings/android/android-build-configuration-options.html#gleanexpirebyversion)

Reply:
I see, this version tells Glean what is the major version.  So with 1 set then no metrics will expire, correct?

Reply:
@rocketsroger correct, but I strongly discourage you from landing it as it is: you should really provide the correct Fenix version in here, within this PR. If you want non-expiring metrics, you can make non-expiring metrics via the metrics.yaml (by requesting the proper data-review).

Reply:
@Dexterp37 yes, I won't land this until I update the patch.  Just want to make sure I understand this correctly.  The version number we pass in here is the version number Glean will compare against the metrics `expires:` field correct?  So if we pass in `ext.gleanExpireByVersion = 99` then anything below 100 will expire but metrics that `expires: 100` should still work.  Correct?

Reply:
`gleanExpireByVersion = 99` means that Fenix is version 99.x.x so all the metrics bound to expire in versions <= 99 will be expired.